### PR TITLE
Travis commit message check is modified

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Tendrl-bug-id:Tendrl/node_agent#86

Signed-off-by: root <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @anmolbabu @shtripat please review it

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat @anmolbabu please review it

**Comment by r0h4n** [comment]
This is still broken, please use commit msg format from https://github.com/Tendrl/common/pull/64 as @anmolbabu suggested

**Comment by GowthamShanmugam** [comment]
@r0h4n @anmolbabu please review it

File: check_commit_msg.py [hunk 1]
```diff
@@ -25,8 +25,8 @@
             print(ex)
 
 if tendrl_bug_id is None:
-    raise Exception("Please add 'tendrl-bug-id:<tendrl_repo>/issue_id' to "
-                    "your commit msg")
+    raise Exception("Please add 'tendrl-bug-id:Tenderl/<tendrl_repo>#issue_id'"
```
Comment (r0h4n) [inline check_commit_msg.py]: Typo at "Tenderl"

**Comment by GowthamShanmugam** [comment]
@r0h4n please review it

**Pull request merged**